helm_deploy/hmpps-interventions-ui/templates/_envs.tpl

```diff
@@ -10,26 +10,20 @@ env:
   - name: CLIENT_ID
     valueFrom:
       secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: CLIENT_ID
+        name: hmpps-auth
+        key: client-id
 
   - name: CLIENT_SECRET
     valueFrom:
       secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: CLIENT_SECRET
-
-  - name: APPINSIGHTS_INSTRUMENTATIONKEY
-    valueFrom:
-      secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: APPINSIGHTS_INSTRUMENTATIONKEY
+        name: hmpps-auth
+        key: client-secret
 
   - name: SESSION_SECRET
     valueFrom:
       secretKeyRef:
-        name: {{ template "app.name" . }}
-        key: SESSION_SECRET
+        name: session
+        key: secret
 
   - name: REDIS_HOST
```

Comment:
_not a blocker_

🙋‍♂️ This will be populated from a secret provisioned by an ElastiCache resource in [our namespace](https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources).
(We don't have this resource yet.)

❓ I wonder if it would make sense to somehow include the purpose in one of these names, so it's immediately visible that we use it for session storage?

Reply:
yeh we can do. these secret names have no bearing on what we call the secrets in AWS, so we can address this when we migrate off k8s secrets?

Reply:
We may need to address it before (but definitely not in this PR 😄) as Kubernetes secrets will stay as a mechanism used in runtime to inject secrets.

When we look at the Kubernetes namespace (runtime) and the codebase, I feel we should be able to reason about how they connect together.

AWS secrets manager values will be invisible in both contexts and to avoid an additional level of mapping in the helm templates we will probably have those values match the ENV var names or the secret names -- so I get the sense we will not completely detach ourselves from these names.